azure-spring-boot-starters/azure-active-directory-spring-boot-starter/README.md

```diff
@@ -28,6 +28,10 @@ If you are using Maven, add the following dependency.
 Open `application.properties` file and add below properties.
 
 ```
+# If property azure.activedirectory.environment is not configured, it will use default value global
```

Comment:
By default, property `azure.activedirectory.environment` is set to `global`. While if you're using Azure China, please set property value to `cn`, like....

Reply:
Readme is updated.